Sources/Security.php

```diff
@@ -552,13 +550,4 @@ public static function kickGuest(): void
 	}
 }
 
-// Export public static functions and properties to global namespace for backward compatibility.
-if (is_callable(__NAMESPACE__ . '\\Security::exportStatic')) {
-	Security::exportStatic();
-}
-
-// Some functions have moved.
-class_exists('SMF\\SecurityToken');
```

Comment:
Ok, but I must admit, I am having a hard time grasping the purpose of this... Maybe I just can't see the forest for the trees here.

Reply:
As with most things that might make one say, "That's odd. What's that for?", this is for backward compatibility support. This particular line ensures that when SMF\Security is loaded, SMF\SecurityToken is loaded at the same time. This is done because some of the backward compatibility functions supplied by SecurityToken used to reside in Security. This line itself was added before I created the section of Subs-Compat where various classes are loaded when Confg::$backward_compatibility is enabled (i.e. the same one where you are now adding the backward compatibility function wrappers). Now that we are centralizing all of that compatibility loaded into one place, this line is an excellent candidate to be moved there—or better yet, to be replaced with wrapper functions to call SecurityToken's various static methods.

Reply:
My intention is to move all required backcompat function to that file. The only responsibility for the trait after that is global variables since I have not thought of a better way to accomplish that. The variable usage from what I have seen so far is nowhere near as prolific as the function usage though so the cleanup after when the time comes to remove it will be far less pervasive.

This should also allow us to remove all of the class_exist calls from Subs-Compat as well since they will be imported with use statements. Or am I missing something?